src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -50,6 +50,11 @@
 				:class="previewImageClass"
 				alt=""
 				:src="defaultIconUrl">
+			<NcProgressBar v-if="showUploadProgress"
+				class="file-preview__progress"
+				type="circular"
+				:size="56"
+				:value="uploadProgress" />
```

Comment:
Maybe it is good to reduce the opacity of the file icon behind or add transparent white layer with the progressBar.

Reply:
Thought about that 👍